src/components/Label/GlowingLabel.tsx

```diff
@@ -19,3 +20,19 @@ export const GlowingLabel: FC<GlowingLabelProps> = ({ children, showGlow }) => (
     {children}
   </div>
 )
+
+export const GlowingLabel: FC<GlowingLabelProps> = ({ children, showGlow }) => (
```

Comment:
To me it looks like we're duplicating those components. Couldn't we create just one that is customizable? We could also export 2 predefined customizations if needed.

Reply:
updated